docs/mhlo-op-coverage.md

```diff
@@ -42,7 +42,7 @@ The table below shows the supported MHLO ops.
 | tuple                 | :heavy_check_mark: | |
 | compare               | :heavy_check_mark: | |
 | **Slice ops**
-| slice                 | :white_check_mark: | Only for 1D or 2D inputs |
+| slice                 | :heavy_check_mark: | |
```

Comment:
Theoretically dimensions are not limited to four.
```suggestion
| slice                 | :white_check_mark: | For 1 <= dim <= 4 |
```

Reply:
I agree. Please adjust the documentation @OliverScherf. If you commit via the GH UI, it might happen that the commit has the wrong author (private email instead of a business address). In the late you need to manually fix that.

Reply:
I just commited locally and force pushed @simon-camp  suggestion.